src/elements/stream_element.js

```diff
@@ -43,7 +43,7 @@ export class StreamElement extends HTMLElement {
       const event = this.beforeRenderEvent
 
       if (this.dispatchEvent(event)) {
-        await nextEventLoopTick()
+        nextRepaint()
```

Comment:
Do we need to `await` this? Should `nextRepaint()` return the `next`-prefixed Promise utilities instead of `await`-ing them?